textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/keywordmatch/KeywordPhraseTest.java

```diff
@@ -271,6 +271,7 @@ public void testWordInMultipleFieldsQueryWithStopWords3() throws Exception {
         for (ITuple tuple : keywordTestConstants.getSampleMedlineRecord()) {
             MedDataWriter.insertTuple(tuple);
         }
+        MedDataWriter.close();
```

Comment:
The name `MedDataWriter` is not following our naming convention.  @qinhuah please change it to `medDataWriter`.